test/helpers/chainTest.ts

```diff
@@ -5,7 +5,8 @@ import { container } from 'tsyringe'
 import createHttpServer from '../../src/server'
 import Indexer from '../../src/lib/indexer'
 
-import Database, { CertificateRow } from '../../src/lib/db'
+import { CertificateRow } from '../../src/lib/db/types'
```

Comment:
DB changes as separate PR from API controller, testing first with initiation and issuance?